samples/debug/ppi_trace/src/main.c

```diff
@@ -79,7 +80,7 @@ static void alarm_callback(const struct device *dev, uint8_t chan_id,
 static void counter_setup(void)
 {
 	int err;
-	const struct device *dev = device_get_binding(RTC_LABEL);
+	const struct device *dev = DEVICE_DT_GET(RTC_NODE);
 
 	__ASSERT(dev, "Sample cannot run on this board.");
```

Comment:
This assertion should be replaced with a call to `device_is_ready`, as now `dev` will always be non-NULL.

Reply:
oops good catch, fixed!